R/toolendmessage.R

```diff
@@ -11,20 +11,6 @@
 #' hierarchy level by 1).
 #' @author Jan Philipp Dietrich
 #' @seealso \code{\link{toolstartmessage}}, \code{\link{vcat}}
-#' @examples
```

Comment:
why did you remove the example?

Reply:
It was exactly the same as the example in toolstartmessage I believe.

Reply:
I see